config/routes/api_platform.yaml

```
@@ -1,4 +1,4 @@
 api_platform:
   resource: .
   type: api_platform
-  prefix: /api
+  prefix: '%api_url_prefix%'
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)